index.js

```diff
@@ -158,6 +158,9 @@ exports.addTasks = (gulp, libraryName, srcGlob, webpackConfig, dtsGlob) => { //e
                        } else {
                            thePath.dirname = path.join('js', path.relative(SRC, thePath.dirname));
                        }
+                       if (thePath.extname === '.ts') {
```

Comment:
Is there reason this is a separate `if` statement? An `else if` construct should be more appropriate